lib/puppet/rest/response_handling.rb

```diff
@@ -0,0 +1,12 @@
+module Puppet::Rest
+  module ResponseHandling
```

Comment:
It looks like we only use this in one place? I think in that case we don't need it in its own module.

This seems like it might be idiomatic to turn into a `#to_error` method on `Response`? Or as a private helper in `Routes`?

Reply:
I tried several configurations of this and wasn't sure where I liked it. I think I'll go with a method on Response.